test/TestMath.sol

```diff
@@ -70,27 +70,25 @@ contract TestMath is Test {
         Math.divUp(x, 0);
     }
 
-    function testDivUpWhenNumSmaller(uint256 x, uint256 y) public {
+    function testDivUpNumSmaller(uint256 x, uint256 y) public {
         vm.assume(x > 0);
         vm.assume(x < y);
         assertEq(Math.divUp(x, y), 1);
     }
 
-    function testDivUpWhenOperandsEqual(uint256 x) public {
+    function testDivUpEqual(uint256 x) public {
         vm.assume(x > 0);
         assertEq(Math.divUp(x, x), 1);
     }
 
-    function testDivUpWhenNumLargerAndDivisible(uint256 x, uint256 y) public {
+    function testDivUpDivisible(uint256 x, uint256 y) public {
         vm.assume(y > 0);
-        vm.assume(x > y);
         vm.assume(x % y == 0);
         assertEq(Math.divUp(x, y), x / y);
     }
 
-    function testDivUpWhenNumLargerAndNotDivisible(uint256 x, uint256 y) public {
```

Comment:
I know test names are large but it makes reading a test report & the code easier by giving a clear context
I don't think it's the case for `testDivUpNotDivisible` which does not indicate whether it's the numerator or the denominator which is not divisible

I would recommend `testDivUpNumNotDivisible` and `testDivUpNumDivisible`, if making test names shorter is meaningful

Reply:
>which does not indicate whether it's the numerator or the denominator which is not divisible
this is implicit

But I can re-add that if you want!

Reply:
>which does not indicate whether it's the numerator or the denominator which is not divisible
this is implicit

But I can re-add that if you want!